tests/jury/metrics/test_cer.py

```diff
@@ -0,0 +1,56 @@
+import pytest
+
+from jury import Jury
+from jury.metrics import AutoMetric
+from tests.jury.conftest import get_expected_output
+from tests.utils import assert_almost_equal_dict
+
+
+@pytest.fixture(scope="module")
+def jury():
+    metric = AutoMetric.load("cer")
+    return Jury(metrics=metric)
+
+
+@pytest.fixture(scope="module")
+def jury_concat():
+    metric = AutoMetric.load("cer", compute_kwargs={"concatenate_texts": True})
+    return Jury(metrics=metric)
+
+
+@pytest.fixture
+@get_expected_output(prefix="metrics")
+def output_basic():
+    return output_basic.output
+
+
+@pytest.fixture
+@get_expected_output(prefix="metrics")
+def output_multiple_ref():
+    return output_multiple_ref.output
+
+
+@pytest.fixture
+@get_expected_output(prefix="metrics")
+def output_multiple_pred_multiple_ref():
+    return output_multiple_pred_multiple_ref.output
+
+
+def test_basic(predictions, references, jury, output_basic):
+    scores = jury(predictions=predictions, references=references)
+    assert_almost_equal_dict(actual=scores, desired=output_basic)
+
+
+def test_basic_concat(predictions, references, jury_concat, output_basic):
```

Comment:
This looks like it compares the result to the ["basic"] instead of ["basic_concat"]. The test passes because they are identical at the moment (which I'm not sure if it is supposed to be or not).